hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFormatReader.java

```diff
@@ -39,6 +39,8 @@
   private final List<HoodieLogFile> logFiles;
   // Readers for previously scanned log-files that are still open
   private final List<HoodieLogFileReader> prevReadersInOpenState;
+  // Key field to read get the key from record
+  private final String keyField;
```

Comment:
Consider renaming to recordKeyField to be more clear as to which key we are referring to in code.

Reply:
fixed.